#!/usr/bin/env python
PACKAGE = "me5413_world"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed_target", double_t, 1, "Default: 0.5[m/s]", 0.5, 0.1, 1.0)

gen.add("track_A_axis", double_t, 1, "Default: 8.0", 8.0, 1.0, 15.0)
gen.add("track_B_axis", double_t, 1, "Default: 8.0", 8.0, 1.0, 15.0)
gen.add("track_wp_num", int_t, 1, "Default: 500", 500, 100, 2000)
gen.add("local_prev_wp_num", int_t, 1, "Default: 10", 10, 1, 20)
gen.add("local_next_wp_num", int_t, 1, "Default: 50", 50, 5, 200)

exit(gen.generate(PACKAGE, "path_publisher_node", "path_publisher"))